### PR TITLE
Add stop script/automation action

### PR DESCRIPTION
**Pull request by frenck**
## Proposed change

Add support for the stop action in the automation editor.

Backend PR: <https://github.com/home-assistant/core/pull/67340>
Backend PR part 2: <https://github.com/home-assistant/core/pull/70109>

CleanShot 2022-04-15 at 19 26 14

## Type of change

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue or discussion:
- Link to documentation pull request: https://github.com/home-assistant/home-assistant.io/pull/22338

## Checklist

- [x] The code change is tested and works locally.
- [x] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [x] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

File: src/panels/config/automation/action/ha-automation-action-row.ts [hunk 1]
```diff
@@ -50,6 +52,8 @@ const OPTIONS = [
   "repeat",
   "choose",
   "device_id",
+  "stop",
+  "error",
```
Comment (balloob) [inline src/panels/config/automation/action/ha-automation-action-row.ts:56]: I would have expected these to be a single UI. "Error" is ambiguous and so I would name it something like "Stop with an error" or something. In that case, both are "Stop"

Comment (frenck) [reply src/panels/config/automation/action/ha-automation-action-row.ts:56]: Alright, should we adjust the YAML version too in that case?

Current:

```yaml
- stop: "Reason"
- error: "Reason"
```

Possible alternative:

```yaml
- stop:
    reason: "Reason"
    error: true
```

Comment (balloob) [reply src/panels/config/automation/action/ha-automation-action-row.ts:56]: Yeah let's do that.

Comment (frenck) [reply src/panels/config/automation/action/ha-automation-action-row.ts:56]: Alright, converting this one to draft to adjust the backend first

Comment (frenck) [reply src/panels/config/automation/action/ha-automation-action-row.ts:56]: Backend change: <https://github.com/home-assistant/core/pull/70109>

Comment (frenck) [reply src/panels/config/automation/action/ha-automation-action-row.ts]: Frontend adjust here: [4922c90](https://github.com/home-assistant/frontend/pull/12299/commits/4922c90e7a6686ea71bd3d35940a8a218248f4fc)

**Review by zsarnett — DISMISSED** [review]

File: src/translations/en.json [hunk 2]
```diff
@@ -2020,7 +2020,12 @@
                   "if": "If",
                   "then": "Then",
                   "else": "Else"
-                }
+                },
+                "stop": {
+                  "label": "Stop",
+                  "stop": "Reason for stopping",
+                  "error": "Stop because of an unexpected error"
+                }
```
Comment (frenck) [inline src/translations/en.json:2028]:
```suggestion
                }
```

**Comment by frenck** [comment]
Removing the `wait for backend` label, at the parent has been merged.

**Review by zsarnett — APPROVED** [review]

**Pull request merged**